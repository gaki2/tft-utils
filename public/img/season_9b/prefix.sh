for file in TFT9_*.png
do
    mv "$file" "${file#TFT9_}"
done