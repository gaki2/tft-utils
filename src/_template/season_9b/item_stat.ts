import { ItemStat } from '../../script/template/itemStatTemplate';

export const item_stat_season_9b: { [key: string]: { [key in ItemStat]?: number } } = {
  TFT_Consumable_NeekosHelp: {},
  TFT_Item_BFSword: { ad: 10 },
  TFT_Item_Bloodthirster: { ad: 20, mr: 20 },
  TFT_Item_BrambleVest: { armor: 70 },
  TFT_Item_ChainVest: { armor: 20 },
  TFT_Item_SparringGloves: { critical: 20 },
  TFT_Item_Deathblade: { ad: 66 },
  TFT_Item_DragonsClaw: { mr: 65 },
  TFT_Item_EmptyBag: {},
  TFT_Item_ForceOfNature: {},
  TFT_Item_ThiefsGloves: { critical: 20, hp: 150 },
  TFT_Item_ThiefsGloves_Empty: {},
  TFT_Item_Shroud: {},
  TFT_Item_FrozenHeart: { armor: 20, mana: 30 },
  TFT_Item_GiantsBelt: { hp: 150 },
  TFT_Item_MadredsBloodrazor: { ad: 30, ap: 20, speed: 10 },
  TFT_Item_GuardianAngel: { ad: 10, armor: 20 },
  TFT_Item_GuinsoosRageblade: { ap: 10, speed: 18 },
  TFT_Item_UnstableConcoction: { mana: 15, critical: 20 },
  TFT_Item_HextechGunblade: { ad: 10, ap: 10 },
  TFT_Item_Chalice: {},
  TFT_Item_InfinityEdge: { ad: 35, critical: 35 },
  TFT_Item_IonicSpark: { ap: 15, mr: 25 },
  TFT_Item_JeweledGauntlet: { ap: 30, critical: 35 },
  TFT_Item_LastWhisper: { ad: 15, speed: 25, critical: 20 },
  TFT_Item_LocketOfTheIronSolari: {},
  TFT_Item_ArchangelsStaff: { mana: 15, ap: 20 },
  TFT_Item_Spatula: {},
  TFT_Item_Quicksilver: { mr: 30, speed: 30, critical: 20 },
  TFT_Item_Morellonomicon: { ap: 25, hp: 150 },
  TFT_Item_NeedlesslyLargeRod: { ap: 10 },
  TFT_Item_NegatronCloak: { mr: 20 },
  TFT_Item_PowerGauntlet: { hp: 150, critical: 20, ap: 10, speed: 25 },
  TFT_Item_RabadonsDeathcap: { ap: 70 },
  TFT_Item_RapidFireCannon: { speed: 35 },
  TFT_Item_RecurveBow: { speed: 10 },
  TFT_Item_RedBuff: { hp: 250, armor: 20 },
  TFT_Item_Redemption: { hp: 150, mana: 15 },
  TFT_Item_RunaansHurricane: { ad: 20, speed: 10, mr: 20 },
  TFT_Item_SeraphsEmbrace: { mana: 20, ap: 25 },
  TFT_Item_SpearOfShojin: { mana: 15, ap: 25, ad: 25 },
  TFT_Item_StatikkShiv: { mana: 15, ap: 20, speed: 20 },
  TFT_Item_GargoyleStoneplate: { armor: 30, mr: 30 },
  TFT_Item_TearOfTheGoddess: { mana: 15 },
  TFT_Item_TitanicHydra: {},
  TFT_Item_TitansResolve: { armor: 20, speed: 10 },
  TFT_Item_WarmogsArmor: { hp: 800 },
  TFT_Item_ZekesHerald: {},
  TFT_Item_Zephyr: {},
  TFT_Consumable_ItemRemover: {},
  TFT_Consumable_ItemReroller: {},
  TFT_Consumable_ShopReroller: {},
  TFT4_Item_OrnnAnimaVisage: { hp: 450, mr: 25 },
  TFT4_Item_OrnnDeathsDefiance: { ad: 10, armor: 30, speed: 25 },
  TFT4_Item_OrnnEternalWinter: { hp: 200, armor: 20 },
  TFT4_Item_OrnnTheCollector: { ad: 20, critical: 30 },
  TFT4_Item_OrnnInfinityForce: {
    hp: 250,
    armor: 25,
    ap: 25,
    mr: 25,
    ad: 25,
    mana: 25,
    critical: 25,
    speed: 25,
  },
  TFT4_Item_OrnnMuramana: { mana: 15, ad: 15, speed: 15, ap: 10 },
  TFT4_Item_OrnnObsidianCleaver: {},
  TFT4_Item_OrnnRanduinsSanctum: {},
  TFT4_Item_OrnnZhonyasParadox: {},
  TFT5_Item_TrapClawRadiant: { hp: 150, speed: 40, ap: 30, critical: 20 },
  TFT5_Item_BloodthirsterRadiant: { ad: 40, mr: 20 },
  TFT5_Item_IonicSparkRadiant: { ap: 15, mr: 25 },
  TFT5_Item_GiantSlayerRadiant: { ad: 50, ap: 40, speed: 10 },
  TFT5_Item_DragonsClawRadiant: { mr: 130 },
  TFT5_Item_GargoyleStoneplateRadiant: { mr: 30, armor: 30 },
  TFT5_Item_LastWhisperRadiant: { ad: 45, speed: 25, critical: 55 },
  TFT5_Item_HandOfJusticeRadiant: { critical: 20, mana: 15 },
  TFT5_Item_FrozenHeartRadiant: { mana: 30, armor: 40 },
  TFT5_Item_JeweledGauntletRadiant: { ap: 55, critical: 75 },
  TFT5_Item_GuardianAngelRadiant: { ad: 30, armor: 30 },
  TFT5_Item_GuinsoosRagebladeRadiant: { ap: 8, speed: 20 },
  TFT5_Item_HextechGunbladeRadiant: { ad: 10, ap: 40 },
  TFT5_Item_DeathbladeRadiant: { ad: 125 },
  TFT5_Item_MorellonomiconRadiant: { hp: 150, ap: 40 },
  TFT5_Item_QuicksilverRadiant: { mr: 40, critical: 40, speed: 50 },
  TFT5_Item_RabadonsDeathcapRadiant: { ap: 140 },
  TFT5_Item_RedemptionRadiant: { mana: 15, hp: 200 },
  TFT5_Item_RapidFirecannonRadiant: { speed: 70 },
  TFT5_Item_ThiefsGlovesRadiant: { critical: 20, hp: 150 },
  TFT5_Item_BrambleVestRadiant: { armor: 140 },
  TFT5_Item_RunaansHurricaneRadiant: { ad: 35, mr: 20, speed: 20 },
  TFT5_Item_SpearOfShojinRadiant: { ad: 45, mana: 15, ap: 45 },
  TFT5_Item_StatikkShivRadiant: { mana: 15, ap: 50, speed: 20 },
  TFT5_Item_SunfireCapeRadiant: { hp: 300, armor: 20 },
  TFT5_Item_TitansResolveRadiant: { armor: 35, speed: 30 },
  TFT5_Item_ArchangelsStaffRadiant: { mana: 15, ap: 50 },
  TFT5_Item_WarmogsArmorRadiant: { hp: 1300 },
  TFT5_Item_InfinityEdgeRadiant: { ad: 65, critical: 75 },
  TFT5_Item_ZzRotPortalRadiant: {},
  TFT7_Item_ShimmerscaleMogulsMail: { hp: 150 },
  TFT7_Item_ShimmerscaleHeartOfGold: {},
  TFT7_Item_ShimmerscaleGoldmancersStaff: { ap: 30 },
  TFT_Consumable_ChampionDuplicator_III: {},
  TFT9_Item_OrnnHullbreaker: { armor: 35, mr: 35, speed: 30 },
  TFT9_Item_OrnnTrickstersGlass: { armor: 15, mr: 15, speed: 15, critical: 15 },
  TFT9_Item_OrnnDeathfireGrasp: { mana: 30, ap: 50 },
  TFT9_Item_OrnnHorizonFocus: { ad: 15, ap: 15, speed: 15 },
  TFT_Item_UnusableSlot: {},
  TFT_Item_BlankSlot: {},
  TFT_Item_SpectralGauntlet: { hp: 150, mr: 20 },
  TFT5_Item_SpectralGauntletRadiant: { hp: 400, mr: 30 },
  TFT_Item_Crownguard: { ap: 20, armor: 20, hp: 100 },
  TFT5_Item_CrownguardRadiant: { ap: 50, armor: 40, hp: 100 },
  TFT7_Item_ShimmerscaleGoldmancersStaff_HR: { ap: 30 },
  TFT_Item_NightHarvester: { ap: 15, ad: 15, armor: 25, critical: 20 },
  TFT7_Item_ShimmerscaleHeartOfGold_HR: {},
  TFT5_Item_NightHarvesterRadiant: { ap: 25, ad: 25, armor: 50, critical: 20 },
  TFT7_Item_ShimmerscaleMogulsMail_HR: {},
  TFT_Item_SteraksGage: { hp: 200, ad: 15 },
  TFT5_Item_SteraksGageRadiant: { hp: 400, ad: 30 },
  TFT_Item_BlueBuff: { mana: 20, ap: 25 },
  TFT5_Item_BlueBuffRadiant: { mana: 20, ap: 50 },
  TFT_Item_AdaptiveHelm: { mana: 15, ap: 15, mr: 20 },
  TFT5_Item_AdaptiveHelmRadiant: { mana: 15, mr: 30 },
  TFT_Item_Leviathan: { hp: 150, ap: 30, speed: 15 },
  TFT5_Item_LeviathanRadiant: { hp: 200, ap: 55, speed: 20 },
  TFT_Item_AegisOfTheLegion: {},
  TFT_Item_RadiantVirtue: {},
  TFT_Item_BansheesVeil: {},
  TFT9_Item_OrnnPrototypeForge: {},
  TFT9_Item_StrategistEmblem: {},
  TFT9_Item_SlayerEmblem: {},
  TFT9_Item_BrawlerEmblem: {},
  TFT9_Item_BastionEmblem: {},
  TFT9_Item_ArmorcladEmblem: {},
  TFT9_Item_DemaciaEmblem: {},
  TFT9_Consumable_GoldenItemRemover: {},
  TFT9_Item_VoidEmblem: {},
  TFT9_Item_IoniaEmblem: {},
  TFT9_Item_PreserverEmblem: {},
  TFT9_Consumable_Chempunk_ChemtankMod: {},
  TFT9_Item_SorcererEmblem: {},
  TFT9_Consumable_Chempunk_BloodRageMod: {},
  TFT9_Consumable_Chempunk_AdaptiveImplantMod: {},
  TFT9_Item_ZaunEmblem: {},
  TFT9_Item_TargonEmblem: {},
  TFT9_Item_RogueEmblem: {},
  TFT9_Item_WorldEnder_DarkinBlade: {},
  TFT9_Item_TrickshotEmblem: {},
  TFT9_Item_ChallengerEmblem: {},
  TFT9_Item_DeadeyeEmblem: {},
  TFT_Assist_Gold_6: {},
  TFT_Assist_Gold_5: {},
  TFT_Assist_Gold_10: {},
  TFT_Assist_Gold_20: {},
  TFT_Assist_Gold_40: {},
  TFT_Assist_Gold_54: {},
  TFT9_Item_PiltoverEmblem: {},
  TFT9_Item_NoxusEmblem: {},
  TFT9_Item_ShurimaEmblem: {},
  TFT9_Consumable_EagleEye_Armor: {},
  TFT9_Consumable_EagleEye_MagicResist: {},
  TFT9_Item_FreljordEmblem: {},
  TFT9_HeimerUpgrade_ShrinkRay: {},
  TFT9_HeimerUpgrade_Goldification: {},
  TFT9_HeimerUpgrade_SelfRepair: {},
  TFT9_HeimerUpgrade_MicroRockets: {},
  TFT9_Item_CrownOfDemacia: {},
  TFT9_Consumable_MasterworkUpgrade: {},
  TFT9_Consumable_ScrollOfTraits: {},
  TFT9_Consumable_Chempunk_UnstableChemtankMod: {},
  TFT9_Consumable_Chempunk_RoboticArmMod: {},
  TFT9_Consumable_Chempunk_VirulentBiowareMod: {},
  TFT9_Item_ShadowIslesEmblem: {},
  TFT9_Item_PiltoverProgress: {},
  TFT9_Item_PiltoverCharges: {},
  TFT_Assist_Gold_7: {},
  TFT_Assist_Gold_8: {},
  TFT7_Item_TreasureDragonGold9: {},
  TFT7_Item_TreasureDragonBlankSlot: {},
  TFT9_Item_CrownOfDemacia_DU: {},
  TFT9_Item_BaronsHead: {},
  TFT7_Item_TreasureDragonGold4: {},
  TFT7_Item_TreasureDragonGold3: {},
  TFT7_Item_TreasureDragonGold20: {},
  TFT_Consumable_TrainingDummy: {},
  TFT9_Item_BilgewaterEmblem: {},
  TFT9_Item_VanquisherEmblem: {},
  TFT9_Item_IxtalEmblem: {},
  TFT9_HeimerUpgrade_RefractorBeam: {},
};
