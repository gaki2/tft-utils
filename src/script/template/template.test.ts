import { describe } from 'vitest';
import { item_stat_season_9b } from '../../_template/season_9b/item_stat';

describe('템플릿 테스트', () => {
  test('[시즌9b] 아이템 스탯 데이터 스냅샷 테스트', () => {
    const item_stat_data = item_stat_season_9b;

    expect(item_stat_data).toMatchSnapshot();
  });
});
