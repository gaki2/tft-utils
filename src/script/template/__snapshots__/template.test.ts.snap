// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`템플릿 테스트 > [시즌9b] 아이템 스탯 데이터 스냅샷 테스트 1`] = `
{
  "TFT4_Item_OrnnAnimaVisage": {
    "hp": 450,
    "mr": 25,
  },
  "TFT4_Item_OrnnDeathsDefiance": {
    "ad": 10,
    "armor": 30,
    "speed": 25,
  },
  "TFT4_Item_OrnnEternalWinter": {
    "armor": 20,
    "hp": 200,
  },
  "TFT4_Item_OrnnInfinityForce": {
    "ad": 25,
    "ap": 25,
    "armor": 25,
    "critical": 25,
    "hp": 250,
    "mana": 25,
    "mr": 25,
    "speed": 25,
  },
  "TFT4_Item_OrnnMuramana": {
    "ad": 15,
    "ap": 10,
    "mana": 15,
    "speed": 15,
  },
  "TFT4_Item_OrnnObsidianCleaver": {},
  "TFT4_Item_OrnnRanduinsSanctum": {},
  "TFT4_Item_OrnnTheCollector": {
    "ad": 20,
    "critical": 30,
  },
  "TFT4_Item_OrnnZhonyasParadox": {},
  "TFT5_Item_AdaptiveHelmRadiant": {
    "mana": 15,
    "mr": 30,
  },
  "TFT5_Item_ArchangelsStaffRadiant": {
    "ap": 50,
    "mana": 15,
  },
  "TFT5_Item_BloodthirsterRadiant": {
    "ad": 40,
    "mr": 20,
  },
  "TFT5_Item_BlueBuffRadiant": {
    "ap": 50,
    "mana": 20,
  },
  "TFT5_Item_BrambleVestRadiant": {
    "armor": 140,
  },
  "TFT5_Item_CrownguardRadiant": {
    "ap": 50,
    "armor": 40,
    "hp": 100,
  },
  "TFT5_Item_DeathbladeRadiant": {
    "ad": 125,
  },
  "TFT5_Item_DragonsClawRadiant": {
    "mr": 130,
  },
  "TFT5_Item_FrozenHeartRadiant": {
    "armor": 40,
    "mana": 30,
  },
  "TFT5_Item_GargoyleStoneplateRadiant": {
    "armor": 30,
    "mr": 30,
  },
  "TFT5_Item_GiantSlayerRadiant": {
    "ad": 50,
    "ap": 40,
    "speed": 10,
  },
  "TFT5_Item_GuardianAngelRadiant": {
    "ad": 30,
    "armor": 30,
  },
  "TFT5_Item_GuinsoosRagebladeRadiant": {
    "ap": 8,
    "speed": 20,
  },
  "TFT5_Item_HandOfJusticeRadiant": {
    "critical": 20,
    "mana": 15,
  },
  "TFT5_Item_HextechGunbladeRadiant": {
    "ad": 10,
    "ap": 40,
  },
  "TFT5_Item_InfinityEdgeRadiant": {
    "ad": 65,
    "critical": 75,
  },
  "TFT5_Item_IonicSparkRadiant": {
    "ap": 15,
    "mr": 25,
  },
  "TFT5_Item_JeweledGauntletRadiant": {
    "ap": 55,
    "critical": 75,
  },
  "TFT5_Item_LastWhisperRadiant": {
    "ad": 45,
    "critical": 55,
    "speed": 25,
  },
  "TFT5_Item_LeviathanRadiant": {
    "ap": 55,
    "hp": 200,
    "speed": 20,
  },
  "TFT5_Item_MorellonomiconRadiant": {
    "ap": 40,
    "hp": 150,
  },
  "TFT5_Item_NightHarvesterRadiant": {
    "ad": 25,
    "ap": 25,
    "armor": 50,
    "critical": 20,
  },
  "TFT5_Item_QuicksilverRadiant": {
    "critical": 40,
    "mr": 40,
    "speed": 50,
  },
  "TFT5_Item_RabadonsDeathcapRadiant": {
    "ap": 140,
  },
  "TFT5_Item_RapidFirecannonRadiant": {
    "speed": 70,
  },
  "TFT5_Item_RedemptionRadiant": {
    "hp": 200,
    "mana": 15,
  },
  "TFT5_Item_RunaansHurricaneRadiant": {
    "ad": 35,
    "mr": 20,
    "speed": 20,
  },
  "TFT5_Item_SpearOfShojinRadiant": {
    "ad": 45,
    "ap": 45,
    "mana": 15,
  },
  "TFT5_Item_SpectralGauntletRadiant": {
    "hp": 400,
    "mr": 30,
  },
  "TFT5_Item_StatikkShivRadiant": {
    "ap": 50,
    "mana": 15,
    "speed": 20,
  },
  "TFT5_Item_SteraksGageRadiant": {
    "ad": 30,
    "hp": 400,
  },
  "TFT5_Item_SunfireCapeRadiant": {
    "armor": 20,
    "hp": 300,
  },
  "TFT5_Item_ThiefsGlovesRadiant": {
    "critical": 20,
    "hp": 150,
  },
  "TFT5_Item_TitansResolveRadiant": {
    "armor": 35,
    "speed": 30,
  },
  "TFT5_Item_TrapClawRadiant": {
    "ap": 30,
    "critical": 20,
    "hp": 150,
    "speed": 40,
  },
  "TFT5_Item_WarmogsArmorRadiant": {
    "hp": 1300,
  },
  "TFT5_Item_ZzRotPortalRadiant": {},
  "TFT7_Item_ShimmerscaleGoldmancersStaff": {
    "ap": 30,
  },
  "TFT7_Item_ShimmerscaleGoldmancersStaff_HR": {
    "ap": 30,
  },
  "TFT7_Item_ShimmerscaleHeartOfGold": {},
  "TFT7_Item_ShimmerscaleHeartOfGold_HR": {},
  "TFT7_Item_ShimmerscaleMogulsMail": {
    "hp": 150,
  },
  "TFT7_Item_ShimmerscaleMogulsMail_HR": {},
  "TFT7_Item_TreasureDragonBlankSlot": {},
  "TFT7_Item_TreasureDragonGold20": {},
  "TFT7_Item_TreasureDragonGold3": {},
  "TFT7_Item_TreasureDragonGold4": {},
  "TFT7_Item_TreasureDragonGold9": {},
  "TFT9_Consumable_Chempunk_AdaptiveImplantMod": {},
  "TFT9_Consumable_Chempunk_BloodRageMod": {},
  "TFT9_Consumable_Chempunk_ChemtankMod": {},
  "TFT9_Consumable_Chempunk_RoboticArmMod": {},
  "TFT9_Consumable_Chempunk_UnstableChemtankMod": {},
  "TFT9_Consumable_Chempunk_VirulentBiowareMod": {},
  "TFT9_Consumable_EagleEye_Armor": {},
  "TFT9_Consumable_EagleEye_MagicResist": {},
  "TFT9_Consumable_GoldenItemRemover": {},
  "TFT9_Consumable_MasterworkUpgrade": {},
  "TFT9_Consumable_ScrollOfTraits": {},
  "TFT9_HeimerUpgrade_Goldification": {},
  "TFT9_HeimerUpgrade_MicroRockets": {},
  "TFT9_HeimerUpgrade_RefractorBeam": {},
  "TFT9_HeimerUpgrade_SelfRepair": {},
  "TFT9_HeimerUpgrade_ShrinkRay": {},
  "TFT9_Item_ArmorcladEmblem": {},
  "TFT9_Item_BaronsHead": {},
  "TFT9_Item_BastionEmblem": {},
  "TFT9_Item_BilgewaterEmblem": {},
  "TFT9_Item_BrawlerEmblem": {},
  "TFT9_Item_ChallengerEmblem": {},
  "TFT9_Item_CrownOfDemacia": {},
  "TFT9_Item_CrownOfDemacia_DU": {},
  "TFT9_Item_DeadeyeEmblem": {},
  "TFT9_Item_DemaciaEmblem": {},
  "TFT9_Item_FreljordEmblem": {},
  "TFT9_Item_IoniaEmblem": {},
  "TFT9_Item_IxtalEmblem": {},
  "TFT9_Item_NoxusEmblem": {},
  "TFT9_Item_OrnnDeathfireGrasp": {
    "ap": 50,
    "mana": 30,
  },
  "TFT9_Item_OrnnHorizonFocus": {
    "ad": 15,
    "ap": 15,
    "speed": 15,
  },
  "TFT9_Item_OrnnHullbreaker": {
    "armor": 35,
    "mr": 35,
    "speed": 30,
  },
  "TFT9_Item_OrnnPrototypeForge": {},
  "TFT9_Item_OrnnTrickstersGlass": {
    "armor": 15,
    "critical": 15,
    "mr": 15,
    "speed": 15,
  },
  "TFT9_Item_PiltoverCharges": {},
  "TFT9_Item_PiltoverEmblem": {},
  "TFT9_Item_PiltoverProgress": {},
  "TFT9_Item_PreserverEmblem": {},
  "TFT9_Item_RogueEmblem": {},
  "TFT9_Item_ShadowIslesEmblem": {},
  "TFT9_Item_ShurimaEmblem": {},
  "TFT9_Item_SlayerEmblem": {},
  "TFT9_Item_SorcererEmblem": {},
  "TFT9_Item_StrategistEmblem": {},
  "TFT9_Item_TargonEmblem": {},
  "TFT9_Item_TrickshotEmblem": {},
  "TFT9_Item_VanquisherEmblem": {},
  "TFT9_Item_VoidEmblem": {},
  "TFT9_Item_WorldEnder_DarkinBlade": {},
  "TFT9_Item_ZaunEmblem": {},
  "TFT_Assist_Gold_10": {},
  "TFT_Assist_Gold_20": {},
  "TFT_Assist_Gold_40": {},
  "TFT_Assist_Gold_5": {},
  "TFT_Assist_Gold_54": {},
  "TFT_Assist_Gold_6": {},
  "TFT_Assist_Gold_7": {},
  "TFT_Assist_Gold_8": {},
  "TFT_Consumable_ChampionDuplicator_III": {},
  "TFT_Consumable_ItemRemover": {},
  "TFT_Consumable_ItemReroller": {},
  "TFT_Consumable_NeekosHelp": {},
  "TFT_Consumable_ShopReroller": {},
  "TFT_Consumable_TrainingDummy": {},
  "TFT_Item_AdaptiveHelm": {
    "ap": 15,
    "mana": 15,
    "mr": 20,
  },
  "TFT_Item_AegisOfTheLegion": {},
  "TFT_Item_ArchangelsStaff": {
    "ap": 20,
    "mana": 15,
  },
  "TFT_Item_BFSword": {
    "ad": 10,
  },
  "TFT_Item_BansheesVeil": {},
  "TFT_Item_BlankSlot": {},
  "TFT_Item_Bloodthirster": {
    "ad": 20,
    "mr": 20,
  },
  "TFT_Item_BlueBuff": {
    "ap": 25,
    "mana": 20,
  },
  "TFT_Item_BrambleVest": {
    "armor": 70,
  },
  "TFT_Item_ChainVest": {
    "armor": 20,
  },
  "TFT_Item_Chalice": {},
  "TFT_Item_Crownguard": {
    "ap": 20,
    "armor": 20,
    "hp": 100,
  },
  "TFT_Item_Deathblade": {
    "ad": 66,
  },
  "TFT_Item_DragonsClaw": {
    "mr": 65,
  },
  "TFT_Item_EmptyBag": {},
  "TFT_Item_ForceOfNature": {},
  "TFT_Item_FrozenHeart": {
    "armor": 20,
    "mana": 30,
  },
  "TFT_Item_GargoyleStoneplate": {
    "armor": 30,
    "mr": 30,
  },
  "TFT_Item_GiantsBelt": {
    "hp": 150,
  },
  "TFT_Item_GuardianAngel": {
    "ad": 10,
    "armor": 20,
  },
  "TFT_Item_GuinsoosRageblade": {
    "ap": 10,
    "speed": 18,
  },
  "TFT_Item_HextechGunblade": {
    "ad": 10,
    "ap": 10,
  },
  "TFT_Item_InfinityEdge": {
    "ad": 35,
    "critical": 35,
  },
  "TFT_Item_IonicSpark": {
    "ap": 15,
    "mr": 25,
  },
  "TFT_Item_JeweledGauntlet": {
    "ap": 30,
    "critical": 35,
  },
  "TFT_Item_LastWhisper": {
    "ad": 15,
    "critical": 20,
    "speed": 25,
  },
  "TFT_Item_Leviathan": {
    "ap": 30,
    "hp": 150,
    "speed": 15,
  },
  "TFT_Item_LocketOfTheIronSolari": {},
  "TFT_Item_MadredsBloodrazor": {
    "ad": 30,
    "ap": 20,
    "speed": 10,
  },
  "TFT_Item_Morellonomicon": {
    "ap": 25,
    "hp": 150,
  },
  "TFT_Item_NeedlesslyLargeRod": {
    "ap": 10,
  },
  "TFT_Item_NegatronCloak": {
    "mr": 20,
  },
  "TFT_Item_NightHarvester": {
    "ad": 15,
    "ap": 15,
    "armor": 25,
    "critical": 20,
  },
  "TFT_Item_PowerGauntlet": {
    "ap": 10,
    "critical": 20,
    "hp": 150,
    "speed": 25,
  },
  "TFT_Item_Quicksilver": {
    "critical": 20,
    "mr": 30,
    "speed": 30,
  },
  "TFT_Item_RabadonsDeathcap": {
    "ap": 70,
  },
  "TFT_Item_RadiantVirtue": {},
  "TFT_Item_RapidFireCannon": {
    "speed": 35,
  },
  "TFT_Item_RecurveBow": {
    "speed": 10,
  },
  "TFT_Item_RedBuff": {
    "armor": 20,
    "hp": 250,
  },
  "TFT_Item_Redemption": {
    "hp": 150,
    "mana": 15,
  },
  "TFT_Item_RunaansHurricane": {
    "ad": 20,
    "mr": 20,
    "speed": 10,
  },
  "TFT_Item_SeraphsEmbrace": {
    "ap": 25,
    "mana": 20,
  },
  "TFT_Item_Shroud": {},
  "TFT_Item_SparringGloves": {
    "critical": 20,
  },
  "TFT_Item_Spatula": {},
  "TFT_Item_SpearOfShojin": {
    "ad": 25,
    "ap": 25,
    "mana": 15,
  },
  "TFT_Item_SpectralGauntlet": {
    "hp": 150,
    "mr": 20,
  },
  "TFT_Item_StatikkShiv": {
    "ap": 20,
    "mana": 15,
    "speed": 20,
  },
  "TFT_Item_SteraksGage": {
    "ad": 15,
    "hp": 200,
  },
  "TFT_Item_TearOfTheGoddess": {
    "mana": 15,
  },
  "TFT_Item_ThiefsGloves": {
    "critical": 20,
    "hp": 150,
  },
  "TFT_Item_ThiefsGloves_Empty": {},
  "TFT_Item_TitanicHydra": {},
  "TFT_Item_TitansResolve": {
    "armor": 20,
    "speed": 10,
  },
  "TFT_Item_UnstableConcoction": {
    "critical": 20,
    "mana": 15,
  },
  "TFT_Item_UnusableSlot": {},
  "TFT_Item_WarmogsArmor": {
    "hp": 800,
  },
  "TFT_Item_ZekesHerald": {},
  "TFT_Item_Zephyr": {},
}
`;
